packages/components/src/InputFile/InputFile.tsx

```diff
@@ -225,18 +230,23 @@ export function InputFile({
     } = await getUploadParams(file);
 
     const fileUpload = getFileUpload(file, key);
-    onUploadStart && onUploadStart({ ...fileUpload });
+    onUploadStart &&
+      onUploadStart({
+        ...fileUpload,
+        uploadUrl: url,
```

Comment:
Would it make sense to add `uploadUrl` to the `fileUpload` returns from `getFileUpload`? Rather than appending on each use?

Reply:
Nevermind you already made that change 😄